opentelemetry-sdk/src/opentelemetry/sdk/trace/__init__.py

```diff
@@ -987,13 +987,12 @@ def record_exception(
         escaped: bool = False,
     ) -> None:
         """Records an exception as a span event."""
-        try:
-            stacktrace = traceback.format_exc()
-        except Exception:  # pylint: disable=broad-except
-            # workaround for python 3.4, format_exc can raise
-            # an AttributeError if the __context__ on
-            # an exception is None
-            stacktrace = "Exception occurred on stacktrace formatting"
+        # TODO: drop first argument after baseline is 3.10
```

Comment:
I think it's actually the opposite, after 3.10 we can just do `trace.format_exception(exception)` based on [this](https://docs.python.org/3.11/library/traceback.html#traceback.print_exception:~:text=Since%20Python%203.10%2C%20instead%20of%20passing%20value%20and%20tb%2C%20an%20exception%20object%20can%20be%20passed%20as%20the%20first%20argument.) correctly

Reply:
Updated